paymentapi/src/main/resources/application-local.properties

```diff
@@ -2,45 +2,42 @@ elasticsearch.service.url=localhost:9200
 elasticsearch.service.user=
 elasticsearch.service.password=
 elasticsearch.service.local.environment=true
-
 kafka.client.authentication.enabled=false
-
 payment.card_token.encryption.password=testest
-
-
 payment_default_notify_url=http://localhost:8085/v1/payment/onlinePayment/notify
 payment_default_return_url=http://localhost:8085/v1/payment/onlinePayment/return
-
 # Paytrail payment (Check payment-experience-api config/ folder for more information how to set it to https)
 paytrail_payment_return_success_url=https://example.test:8285/v1/payment/paytrailOnlinePayment/return/success
 paytrail_payment_return_cancel_url=https://example.test:8285/v1/payment/paytrailOnlinePayment/return/cancel
 paytrail_payment_notify_success_url=https://example.test:8285/v1/payment/paytrailOnlinePayment/notify/success
 paytrail_payment_notify_cancel_url=https://example.test:8285/v1/payment/paytrailOnlinePayment/notify/cancel
-
+# Paytrail card url
+paytrail_card_redirect_success_url=https://example.test:8285/v1/payment/paytrailCard/redirect/success
+paytrail_card_redirect_cancel_url=https://example.test:8285/v1/payment/paytrailCard/redirect/cancel
+paytrail_card_callback_success_url=https://example.test:8285/v1/payment/paytrailCard/callback/success
+paytrail_card_callback_cancel_url=https://example.test:8285/v1/payment/paytrailCard/callback/cancel
+# Paytrail update card url
+paytrail_update_card_redirect_success_url=https://example.test:8285/v1/payment/paytrailCardUpdate/redirect/success
+paytrail_update_card_redirect_cancel_url=https://example.test:8285/v1/payment/paytrailCardUpdate/redirect/cancel
+paytrail_update_card_callback_success_url=https://example.test:8285/v1/payment/paytrailCardUpdate/callback/success
+paytrail_update_card_callback_cancel_url=https://example.test:8285/v1/payment/paytrailCardUpdate/callback/cancel
 # Paytrail card redirect urls
 paytrail_card_redirect_success_url=https://example.test:8285/v1/payment/paytrailCard/redirect/success
 paytrail_card_redirect_cancel_url=https://example.test:8285/v1/payment/paytrailCard/redirect/cancel
 paytrail_card_callback_success_url=https://example.test:8285/v1/payment/paytrailCard/callback/success
 paytrail_card_callback_cancel_url=https://example.test:8285/v1/payment/paytrailCard/callback/cancel
-
 # Paytrail refund
 paytrail_refund_success_url=https://webhook.site/461daf2e-102c-4059-b175-34bdab10ea07/v1/payment/paytrailOnlineRefund/success
 paytrail_refund_cancel_url=https://webhook.site/461daf2e-102c-4059-b175-34bdab10ea07/v1/payment/paytrailOnlineRefund/cancel
-
 serviceconfiguration.url=http://host.docker.internal:8187/serviceconfiguration/
 #server.port=8081
-
 logging.level.org.springframework.data.elasticsearch.client.WIRE=trace
-
 spring.activemq.broker-url=failover:(tcp://localhost:61616)
 spring.activemq.user=jmx
 spring.activemq.password=password
 spring.activemq.in-memory=false
-
```

Comment:
Do not remove these spaces, they are there to keep some space between other values that are not in same "context"